### PR TITLE
Jan 21

### PR DESCRIPTION
**Pull request by DanielTurner**
* C4A-732 - Food name - Help guide defects
* C4A-380 - Date mark conditional formatting not following FRS
* C4A-434 - Ingredients hyperlinks don't open help guide
* C4A-416 - Hyperlinks in use by and best before alert boxes do not work
* C4A-440 - Your label heading and no description provided should not show when data is empty
* C4A-421 - Storage and use, Needs a full stop between free text fields
* C4A-424 - Optional textx boxes don't have required validation rules manditory question marked, character limits
* C4A-422 - The links within the Help guide that take you to a different section of the Help guide do not open the new section
* C4A-429- 2 factor confirmation' on cancel button should wipe user progress/cached data - currently does not
* C4A-442 - Summary page - Product sheet details on summary page are not responsive for mobile phone (iphone Safari)
* C4A-425 - Health star rating hyperlink text is not to requirements
* C4A-441 - Summary page - Example label details on summary page are not responsive for mobile phone (iphone Safari)
* C4A-426 - Statements - Images broken
* C4A-443 -  Date marks: the output of answering date mark questions cannot be manually selected/over-ridden
* C4A-427 - Defect #9670 - Ingredients: Arrow icon not centred in text
* C4A-448 - Defect #9715 - Your label: warning note should not be included on the example label
* C4A-428 - Defect #9679 - Incorrect hyperlink text
* C4A-430 - Food name: help guide content defects
* C4A-464 - Product sheet & example label should show blank values if no data entered, should not show generic
* C4A-466 - DATE MARKS & INGREDIENTS: Hyperlinks in the Help guide that link to other sections of the Help guide do not open the intended section
* C4A-417 - Print screen distorts the yes/no buttons
* C4A-411 - Update validation code to ES5 friendly code
* C4A-420 - Update Form.io template code to be ES5 (IE11) compatible
* C4A-449 - Alter menu functionality.
* C4A-362 - Alter menu functionality.

* Turned tests back on
* Additional code for deployment from development repo with CI for deployment and uat branches. *In progress*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,8 +36,7 @@
     "format:partials": "npm run formatFile './src/components/partials/*.js'",
     "lint": "npm run lint:eslint && npm run lint:prettier",
     "format": "npm run format:eslint && npm run format:prettier && npm run format:partials",
-    "test2": "karma start --coverage",
-    "test": "echo hello",
+    "test": "karma start",
```
Comment (duttonw) [inline package.json:39]: no coverage?

File: src/validations/statements/alcohol.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
```
Comment (duttonw) [inline src/validations/statements/alcohol.js:1]: does this slot into form.io validation area?

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests and deploy to binary repo
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Publish
+
+on:
+  push:
+    branches: [develop, uat]
```
Comment (duttonw) [inline .github/workflows/publish.yml:8]: why not on master also? what about tags?

File: .github/workflows/publish.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Archive artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: |
+            dist
+      - name: Publish
+        uses: qld-gov-au/gha-publish-to-git@master
+        with:
+          repository: ${{ secrets.TARGET_REPO }}
```
Comment (duttonw) [inline .github/workflows/publish.yml:47]: nice way on making it configurable 👍

**Review by duttonw — APPROVED** [review]

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**